prog/mutation.go

```diff
@@ -209,7 +209,7 @@ func (p *Prog) Mutate(rs rand.Source, ncalls int, ct *ChoiceTable) {
 // predicate pred.  It iteratively generates simpler programs and asks pred
 // whether it is equal to the orginal program or not. If it is equivalent then
 // the simplification attempt is committed and the process continues.
-func Minimize(p0 *Prog, callIndex0 int, pred func(*Prog, int) bool) (*Prog, int) {
+func Minimize(p0 *Prog, callIndex0 int, pred func(*Prog, int) bool, full bool) (*Prog, int) {
```

Comment:
Please rename 'full' to 'crash'.
I really can't figure out if full means crash or non-crash. When do we want to minimize fuller -- when minimizing a crash or a program giving new coverage? As we discussed, there are minimization strategies that are useful only for crashes, and some that are useful only for new interesting programs.